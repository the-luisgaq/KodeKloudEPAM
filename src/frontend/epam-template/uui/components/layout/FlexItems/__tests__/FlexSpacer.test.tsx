import React from 'react';
import { FlexSpacer } from '../';
import { render } from '@epam/uui-test-utils';

describe('FlexSpacer', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<FlexSpacer />);
        expect(asFragment()).toMatchSnapshot();
    });
});
