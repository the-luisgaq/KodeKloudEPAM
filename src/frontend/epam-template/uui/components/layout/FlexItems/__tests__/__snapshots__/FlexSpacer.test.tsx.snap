// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexSpacer should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="flexSpacer"
  />
</DocumentFragment>
`;
